The --enable-staging flag was enabling *all* games, regardless of what
$enable_omitgames contained.

---
--- configure.in
+++ configure.in
@@ -60,6 +60,18 @@
   gamelist="$allgames"   
 fi
 
+## This is the staging area
+## These games aren't quite ready for prime-time
+
+AC_MSG_CHECKING([whether to enable staging games])
+AC_ARG_ENABLE([staging],
+  [AS_HELP_STRING([--enable-staging],[whether to enable beta game support (default: disabled)])],[],[])
+AC_MSG_RESULT([$enable_staging])
+
+if test "$enable_staging" = "yes"; then
+	allgames="${allgames} ${staginggames}"
+fi
+
 ## This has a particularly silly name because autoconf doesn't allow 
 ## --disable-foo to have arguments so we have to use an --enable-fubar form.
 AC_ARG_ENABLE(omitgames,
@@ -90,18 +102,6 @@
    gamelist=${allgames}
 fi
 
-## This is the staging area
-## These games aren't quite ready for prime-time
-
-AC_MSG_CHECKING([whether to enable staging games])
-AC_ARG_ENABLE([staging],
-  [AS_HELP_STRING([--enable-staging],[whether to enable beta game support (default: disabled)])],[],[])
-AC_MSG_RESULT([$enable_staging])
-
-if test "$enable_staging" = "yes"; then
-	gamelist="${gamelist} ${staginggames}"
-fi
-
 AC_MSG_RESULT([$gamelist])
 
 AC_SUBST([gamelist])
